doc/README_Source.md

```diff
@@ -176,7 +176,7 @@ The following settings are used to configure the Cosmos DB Kafka Source Connecto
 | connect.cosmosdb.containers | string | list of collections to monitor |  Required |
 | connect.cosmosdb.messagekey.enabled | boolean | set if the Kafka message key should be set. Default is `true` | Required |
 | connect.cosmosdb.messagekey.field | string | use the field's value from the document as the message key. Default is `id` | Required |
-| connect.cosmosdb.changefeed.startFromBeginning | boolean |  set if the change feed should start from beginning. Default is `true` | Required |
+| connect.cosmosdb.offset.useLatest | boolean |  Set to `"true"` to use the latest (most recent) source offset, `"false"` to use the earliest recorded offset. Default is `false` | Required |
```

Comment:
will setting to false be the same as starting from the beginning?

Reply:
@marcelopio please give this a look over and ensure that this does not undo / change the behaviour that you added in terms of "start from beginning". this work is intended to address the other issue you raised about potentially losing items if the connector fails before flushing its buffer to Kafka by using the offset capabilities built in to the changefeed lease container.

Reply:
yea false will be equivalent to start from beginning. I can change the variable to `useEarliest` if that helps, but I figured useLatest was a bit more intuitive when talking about offsets.

Reply:
I liked this change a lot, I didn't have the time to test it out, but it seems that it will not change the behaviour of the startFromBeginning.

Reply:
@sivamu, nah i think what you've called the car makes sense. i was just checking